shared/vcjs/verifyCredential.ts

```diff
@@ -32,66 +33,69 @@ export async function verifyCredential(
   verifiableCredential: Credential,
 ): Promise<VerificationResult> {
   try {
-    let purpose: PublicKeyProofPurpose | AssertionProofPurpose;
-    const proof = verifiableCredential.proof;
-    switch (proof.proofPurpose) {
-      case ProofPurpose.PublicKey:
-        purpose = new PublicKeyProofPurpose();
-        break;
-      case ProofPurpose.Assertion:
-        purpose = new AssertionProofPurpose();
-        break;
-    }
-
-    let suite: Ed25519Signature2018 | RsaSignature2018;
-    const suiteOptions = {
-      verificationMethod: proof.verificationMethod,
-      date: proof.created,
-    };
-    switch (proof.type) {
-      case ProofType.ED25519: {
-        suite = new Ed25519Signature2018(suiteOptions);
-        break;
+    //ToDo - Have to remove else part once Vc Verifier Library is built for Swift
+    if (isAndroid()) {
+      let vcVerifierResult = await vcVerifier.verifyCredentials(
+        JSON.stringify(verifiableCredential),
```

Comment:
@KiruthikaJeyashankar need to check this also as for mDoc/mDL credential will be CBOR. JSON.stringify might cause the issue.

Reply:
Sure Noted

Reply:
Added condition for stringifying